### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by HaidyLopp** [MycroftAI/mycroft-core#1358]
bash dev_setup.sh
Installing packages...
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package python-gobject-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python-gobject-2-dev

E: Package 'python-gobject-dev' has no installation candidate

I have installed "python-gobject-2-dev" using : apt-get install python-gobject-2-dev 

But I always get the same error

**Edit** System : Kali GNU/Linux Rolling 64-bit

**Comment by penrods** [comment]
We need more info to help -- what OS and version are you using?  "Linux" is a many-faceted thing!

**Comment by el-tocino** [comment]
The dev_setup.sh calls for it to be installed:
 $SUDO apt-get install -y (stuff, things) python-gobject-dev (other stuff, things)

Debian lists python-gobject-2 as deprecated:
https://packages.debian.org/unstable/python-gobject 

Kali was based on debian at one point, and the packages from debian more than likely work.  Not a great solution but for now, perhaps you could edit the dev_setup.sh script to remove that package, and try installing it from a debian repo?

**Comment by bretsky84** [comment]
I am having this same issue with Debian Testing, there is no package "python-gobject-dev" currently in the Testing repositories.

**Comment by forslund** [comment]
Seems like `python-gobject-dev` is a transitional package installing `python-gobject-2-dev` on stretch so it might work just changing the dependency. On jessie (which is the base OS for Mark-1) it installs a couple of other packages as well so we'll need to test properly before any change. 

Feel free to help by modifying the dev_setup script to install `python-gobject-2-dev` and check if that's enough to solve your issues.

**Comment by doctorgl** [comment]
Perhaps this will help you. I'm using the latest Kali Rolling distribution and had to make a few minor adjustments to the install.

The Kali repositories don't have the s3cmd package, so I added the debian repository  into /etc/apt/sources.list (deb http://ftp.us.debian.or/debian stable main contrib)

The python-gobject-dev package is listed as deprecated and replaced by python-gobject-2-dev, so I changed the one mention of python-gobject-dev in the dev_setup script to python-gobject-2-dev

Everything installed fine after these minor tweaks. The only package pulled from the debian repository was s3cmd, everythign else came from Kali's repositories.
